style_editor.html

```diff
@@ -9340,6 +9742,15 @@
   FIND("VARIANT_VALUE").value = newValue;
   FIND("VARIANT_SLIDER").value = newValue;
   console.log("Updated Variant: " + newValue);
+  clearTimeout(variantTimeoutId);
+  // console.log("BC-Variant value changed: " + newValue);
+  // Variant cookie save delayed so rapid change doesn't do a bunch of saves
+  if (!isNaN(newValue)) {
+    variantTimeoutId = setTimeout(function() {
+      // console.log("BC-Saving cookie variantValue: " + newValue);
+      saveCookie("variantValue", newValue);
```

Comment:
I don't think we should save the variant.
The variant kind of goes with the preset, so if you change the style, then I wouldn't expect to keep the variant.